### DISCUSSION TITLE
Go 语言数据库的实现原理 | Go 语言设计与实现 · /golang-database-sql

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#246] [Announcements]
https://draveness.me/golang/docs/part4-advanced/ch09-stdlib/golang-database-sql/

 Go 语言数据库的实现原理

**Comment by lixianyang** [comment]
通读了一遍，有个整体印象，后面每个部分再细刷一遍。感谢博主大大

**Comment by wujunze** [comment]
适配器模式

**Comment by HookTech** [comment]
还会有文章更新吗？

**Comment by draveness** [comment]
> 还会有文章更新吗？

这个版本所有章节基本都写完了，进入 Review 阶段了，随着 Go 语言的演进，不排除未来会有其他更新

**Comment by zou-weidong** [comment]
终于看完了，感谢博主

**Comment by ultramarin4** [comment]
看罢只觉意犹未尽！
大佬有文章讲讲gRPC的吗

**Comment by wojoin** [comment]
博客看完了，书也看完了。下面就去实践了

**Comment by gengzongjie** [comment]
看完了全部章节，非常感谢！也非常赞，太厉害了！

**Comment by s-khaon** [comment]
看完啦，感谢博主！另外，是时候更新泛型了。

**Comment by 0xMario27** [comment]
读完了打个卡。基本每个都详细的读了。但有个别的因为自己能力尚且有限。深入也没太读懂。写项目再二刷。

**Comment by kiqi007** [comment]
楼主牛逼～

**Comment by karellincoln** [comment]
感谢博主，写的质量很高。很多还不太理解，后面再二刷

**Comment by northes** [comment]
很好的文章，跟着梳理了一遍，后续自己去看的时候就有一个轮廓了，感谢作者 👍
